### DISCUSSION TITLE
[New Source] Apache Druid

### DISCUSSION THREAD

**Discussion by mingfang** [airbytehq/airbyte#33175] [New Connector Request]
https://druid.apache.org

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368234736516) by [Unito](https://www.unito.io)

**Comment by sherifnada** [comment]
@mingfang Thanks for creating this issue! We'll prioritize Druid in our roadmap. Would you mind sharing the destinations you'd be interested in sending data from Druid into?

**Comment by sherifnada** [comment]
And also, what volume of data do you expect to be handling?

**Comment by mingfang** [comment]
I think the volume of data going in to Druid,as a destination, will be much higher than as a source.
I ingest data into Postgres. Do some transformation and then send them to Druid.
Currently the volume will be millions of records or about 1GB per day.

**Comment by grishick** [comment]
Separate issue for Druid destination: https://github.com/airbytehq/airbyte/issues/2538

[a comment by an automated account is not shown]
